Extensions/PlatformBehavior/tests/PlatformerTestHelper.js

```diff
@@ -0,0 +1,229 @@
+
```

Comment:
Because this file will add global variables, I wonder if we should at least change the name of `makeTestRuntimeScene`? To avoid conflicts/make it clear this is not a common util to be reused in other tests?